src/components/Features/Transfer/Transfer.js

```diff
@@ -90,9 +90,9 @@ export const Transfer = () => {
       errorMsg = tooManyDigitsErrorMsg;
     } else if (isNegative(amount)) {
       errorMsg = negativeValueErrorMsg;
-    } else if (amount > balance) {
+    } else if (Number(amount) > Number(balance)) {
```

Comment:
`decimals` 4 lines above is number or string?

Reply:
Number, it comes from the tokens config.

Reply:
so you should wrap the amount with Number, else the compression might be wrong (string > number). no?

Reply:
ah afterDecimal is number.ok